dashboards-observability/common/types/explorer.ts

```diff
@@ -92,6 +92,7 @@ export interface IExplorerProps {
   history: History;
   notifications: NotificationsStart;
   savedObjectId: string;
+  curSelectedTabId : React.MutableRefObject<undefined>;
```

Comment:
how come the type of the object here is `undefined`?

Reply:
It's a string basically but the system suggested using this type as this is the type it detected being passed in the log explorer